lib/PackageChange.js

```diff
@@ -121,17 +121,20 @@ class PackageChange {
     }
     const excludeMatches = micromatch([this.name], excludePatterns);
     const isExcluded = excludeMatches.length > 0;
+    const depChanges = this.getPrintableDependencyChanges();
+    if (hasPrinted || depChanges.length === 0) {
```

Comment:
after reviewing my code to understand what I was writing, I realized this change would skip printing a valid change if that change had no dependencies that changed. For example, if it was just lodash 1 -> 2 (but no deps), it wouldn't get printed.